### PR TITLE
Remove headers from signature verification if gitlab invocation method

### PR DESCRIPTION
**Pull request by omby8888**
# Description

What - Remove headers from signature verification if gitlab invocation method
Why - Gitlab invocation method doesn't send other headers than port's headers 

## Type of change

Please leave one option from the following and delete the rest:

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Non-breaking change (fix of existing functionality that will not change current behavior)
- [ ] Documentation (added/updated documentation)

### CODE REVIEW

File: app/invokers/webhook_invoker.py [hunk 1]
```diff
@@ -295,8 +297,12 @@ def validate_incoming_signature(self, msg: dict) -> bool:
             return False
 
         # Remove the headers to avoid them being used in the signature verification
```
Comment (Tankilevitch) [inline app/invokers/webhook_invoker.py:299]: lets update the comment

**Review by Tankilevitch — CHANGES_REQUESTED** [review]
I would rather add CONTRIBUTING.md on how to test it with environment variables

File: app/core/config.py [hunk 2]
```diff
@@ -11,6 +12,8 @@
     validator,
 )
 
+load_dotenv()
+
```
Comment (Tankilevitch) [inline app/core/config.py:16]: I would do it if specific `env` variable is specified . e.g. `if ENV_RUNTIME='DEBUG'`

File: app/invokers/webhook_invoker.py [hunk 3]
```diff
@@ -313,7 +320,7 @@ def invoke(self, msg: dict, invocation_method: dict) -> None:
         logger.info("WebhookInvoker - start - destination: %s", invocation_method)
         run_id = msg["context"].get("runId")
 
-        if not self.validate_incoming_signature(msg):
+        if not self.validate_incoming_signature(msg, invocation_method['type']):
```
Comment (matan84) [inline app/invokers/webhook_invoker.py:323]: All invocation types have a `type` property?

**Review by matan84 — CHANGES_REQUESTED** [review]
left 1 comment

File: app/invokers/webhook_invoker.py (hunk 3 above)
Comment (omby8888) [reply app/invokers/webhook_invoker.py]: yeah but added just in case 👍

**Review by matan84 — APPROVED** [review]
LGTM

**Review by Tankilevitch — APPROVED** [review]
love that, great job with the `CONTRIBUTING.md`

**Pull request merged**